esp/templates/survey/answers.html

```diff
@@ -59,6 +54,28 @@
                     <li>Average for all classes: {{ q.question.global_average }}</li>
                 {% endif %}
             </ul>
+            <!-- Creates a key that provides details about the Histogram Labeling-->
+            <b>Response Labels:</b>
+            {% for label in q.question.get_params.list %}
+            <ul>
+                <li> <b> {{ forloop.counter }}: </b>{{ label }} </li>
+            </ul>
+            {% endfor %}
+                {% with q.question.get_params as params %}
```

Comment:
This `with` could probably be expanded around the loop above so we are only calling `get_params` once instead of twice.